lib/database_connection.rb

```diff
@@ -0,0 +1,11 @@
+require "pg"
+
+class DatabaseConnection
+  def self.setup(dbname)
+    @connection = PG.connect(dbname: dbname)
+  end
+
+  def self.query(sql, params = [])
+    @connection.exec_params(sql, params)
+  end
+end
```

Comment:
I like extracting the database connection into its own class, following your Single Responsibility Principle